hack/verify-crd-gen.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCHED_PLUGIN_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${SCHED_PLUGIN_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+cd "${SCHED_PLUGIN_ROOT}"
+
+CRD_OPTIONS="crd:trivialVersions=true,preserveUnknownFields=false"
+
+# Download controller-gen locally
+CONTROLLER_GEN="$(shell pwd)/bin/controller-gen"
+$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0)
+
+# Create a temp directory
+_tmpdir="$(mktemp -d "${SCHED_PLUGIN_ROOT}/_tmp")"
+_crdtmp="${_tmpdir}"
+# Generate CRD
+pushd "${_crdtmp}" > /dev/null 2>&1
```

Comment:
nit: All occurrences of `2>&1` can be shortened to `&>`:

```shell
pushd "${_crdtmp}" &> /dev/null
```

Reply:
Updated `2>&1` as suggested.